internal/cluster/sbd.go

```diff
@@ -38,7 +38,7 @@ type SBDDevice struct {
 
 type SBDDump struct {
 	Header          string `mapstructure:"header,omitempty"`
-	Uuid            string `mapstructure:"uuid,omitempty"`
+	UUID            string `mapstructure:"uuid,omitempty" json:"Uuid"`
```

Comment:
I think we don't use `mapstructure` anymore, we could remove it.
About the `json` field, we should check if this can affect somehow in the server side code...
Most probably now, because we don't use them, but this `SnakeCase` things looks suspicious

Reply:
Same as above, it's for mantaining the compatibility with the fixtures, the fixture expect Uuid in that casing, so I changed and linted in the go code, but I had to mantain the compatibility with json, it's strange and we should fix it, but it's another pr scope imho

Reply:
Let's at least be careful on this changes, as this data might be consumed by the web server side